lib/suspenders/app_builder.rb

```diff
@@ -366,6 +372,20 @@ def create_heroku_apps(flags)
       create_production_heroku_app(flags)
     end
 
+    def create_heroku_pipeline
+      pipeline_name = app_name.dasherize
+
+      run_heroku "pipelines:create -a #{pipeline_name}", "staging"
+      run_heroku(
+        "pipelines:add -a #{app_name.dasherize}-staging #{pipeline_name}",
+        "staging",
+      )
+      run_heroku(
+        "pipelines:add -a #{app_name.dasherize}-production #{pipeline_name}",
```

Comment:
Both `app_name.dasherize` here and one line 380 could use `pipeline_name` for the interpolation.
And if on that thought, maybe consider renaming `pipeline_name` var to something that suits all 3 occurrences?